web/packages/teleport/src/components/ToolTipNoPermBadge/ToolTipNoPermBadge.story.tsx

```diff
@@ -40,6 +40,28 @@ export const NoPermissionBadgeComp = () => (
   </SomeBox>
 );
 
+export const EnterpriseOnly = () => (
+  <SomeBox>
+    I'm a sample container
+    <ToolTipNoPermBadge
+      badgeTitle={BadgeTitle.LackingEnterpriseLicense}
+      children={'Lacking enterprise license'}
+    />
+  </SomeBox>
+);
+
+export const StickyTooltip = () => (
+  <SomeBox>
+    I'm a sample container
+    <ToolTipNoPermBadge
+      sticky={true}
+      children={
+        <Box p={3}>I will only disappear after this component is unhovered</Box>
+      }
+    />
```

Comment:
Any reason for the `children` prop over this?

```suggestion
    <ToolTipNoPermBadge
      sticky={true}
    >
        <Box p={3}>I will only disappear after this component is unhovered</Box>
    </ToolTipNoPermBadge>    
```

Reply:
Nothing in particular, I actually started with just that but later found other stories(on the same page for `ToolTipNoPermBadge` component) using `children`. So for consistency, I used it here as well.